rdkit-stubs/Chem/rdchem.pyi

```diff
@@ -143,7 +143,7 @@ class Atom:
     @classmethod
     def SetIsAromatic(cls, RDKit, bool) -> Any: ...
     @classmethod
-    def SetIsotope(cls, RDKit, unsignedint) -> Any: ...
+    def SetIsotope(cls, arg1: int) -> None: ...
```

Comment:
Maybe these are not classmethods, but instead normal methods that take self.

Reply:
Switched over to instance methods.